frontend/src/components/Search/Search.jsx

```diff
@@ -25,12 +25,12 @@ export default function Search() {
                         onChange={(event) => setSearchInput(event.target.value)}
                     />
                     <button type="submit">
-                        <i className="material-icons">search</i>
+                        <i className="fa fa-search"></i>
                     </button>
                 </form>
                 <SearchQuery searchInput={searchInput} />
             </div>
-            <SearchTable />
+            <StockCarousel />
```

Comment:
good example of how having a component serve a single purpose makes it simpler to swap/replace, like the change you've made in this commit.